docs/man.rst

```diff
@@ -26,178 +26,154 @@ To install the Quipucords server and command line interfaced (CLI) client:
 
-*PostgreSQL*
+Note that in log information for the ``qpc-tools cli install`` command, references to ``quipucords server`` are relevant to the Quipucords server, and references to ``qpc CLI`` are relevant to the Quipucords command line interface client.
 
-#. Create the PostgreSQL image TAR file with Docker. Use the the following command, where the package name is ``postgres.9.6.10.tar``:
+**qpc-tools cli install** [**-h**]
+                         [**--offline**]
+                         [**--offline-files** *OFFLINE_FILES*]
+                         [**--version** *SERVER_VERSION*]
+                         [**--home-dir** *HOME_DIR*]
+                         [**--server-host** *SERVER_HOST*]
+                         [**--server-port** *SERVER_PORT*]
 
-   ``docker pull postgres:9.6.10 && docker save -o postgres.9.6.10.tar postgres:9.6.10``
+``--offline``
 
-*qpc tools command line interface*
+  Controls whether the installation runs as an offlien (disconnected) installation.
 
-#. Go to the following URL: https://github.com/quipucords/qpc/releases
+``--offline-files=OFFLINE_FILES``
 
-#. Download the package that is applicable to the operating system version:
-   - Red Hat Enterprise Linux 6 and CentOS 6: ``qpc.el6.noarch.rpm``
-   - Red Hat Enterprise Linux 7 and CentOS 7: ``qpc.el7.noarch.rpm``
+  Sets the fully qualified path to the files needed to complete an offline installation. Required if ``offline`` specified.
 
-Setting the package location
-~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+``--version=VERSION``
 
-#. Create a packages directory the following paths. For the variable marked as ``{lib}``, enter the library version, either lib or lib64. For the variable marked as ``{x.y.z}``, enter the version of the qpc-tools:
+  Enables the installation of a specific Quipucords CLI version. Contains the semantic versioning format (version.release.patch, such as 0.9.0) of the Quipucords CLI that you want to install.
 
-   ``mkdir -p /usr/{lib}/qpc-tools-{x.y.z}/install/packages``
+``--home-dir=HOME_DIR``
 
-#. Move the packages to the following directory so that the install command can find them:
+  Sets the fully qualified path to the installation directory for the Quipucords CLI. Defaults to ``~/quipucords/``.
 
-   ``mv path/to/quipucords_server_image.tar.gz /usr/{lib}/qpc-tools-{x.y.z}/install/packages``
+``--server-host=SERVER_HOST``
 
-Running the offline installation
-~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
-To complete an installation on a machine without internet connectivity, also known as an offline installation, run the ``qpc-tools`` command with the appropriate options. For example, if you are installing version 0.9.0 of the Quipucords server and command line interface, you would enter the following command:
+  Sets the host for the Quipucords server. Defaults to ``127.0.0.1``.
 
-``qpc-tools install -e install_offline=true -e server_version=0.9.0  -e cli_version=0.9.0``
+``--server-port=SERVER_PORT``
```

Comment:
for the server we say `port` and for the cli we say `server-port` - I assume that we removed server for redundancy in the server but it might be odd to have this inconsistency.

Reply:
Yes, it was removed due to redundancy.

Reply:
Okay 👍